backend/de.metas.business/src/main/java/de/metas/project/workorder/undertest/WorkOrderProjectObjectUnderTestRepository.java

```diff
@@ -97,6 +107,19 @@ public List<WOProjectObjectUnderTest> updateAll(@NonNull final Collection<WOProj
 			record.setWOObjectName(objectUnderTest.getWoObjectName());
 			record.setWOObjectWhereabouts(objectUnderTest.getWoObjectWhereabouts());
 
+			if (objectUnderTest.getProductId() != null)
+			{
+				record.setM_Product_ID(objectUnderTest.getProductId().getRepoId());
+			}
+			if (objectUnderTest.getObjectDeliveredDate() != null)
+			{
+				record.setObjectDeliveredDate(TimeUtil.asTimestamp(objectUnderTest.getObjectDeliveredDate()));
+			}
+			if (objectUnderTest.getOrderLineProvisionId() != null)
+			{
+				record.setC_OrderLine_Provision_ID(OrderLineId.toRepoId(objectUnderTest.getOrderLineProvisionId()));
+			}
```

Comment:
not sure why `externalId` is only persisted if `!= null` but we don't want that logic in standard `Repository.update` method, thus, pls always persist the value received for `productid`, `ObjectDeliveredDate`, `OrderLineProvisionId`